rcgen/src/certificate.rs

```diff
@@ -157,7 +157,7 @@ impl CertificateParams {
 	pub fn signed_by(
 		self,
 		public_key: &impl PublicKeyData,
-		issuer: &impl AsRef<Self>,
+		issuer: &impl AsRef<CertificateParams>,
```

Comment:
Nit: let's stick to `Self` here?

Reply:
Hmm, I tried to make an explanation of this change in the PR summary.

Reply:
The argument you made seems reasonable to me in absence of a reason to prefer keeping it as-is.